sdk/core/azure-core/src/main/java/com/azure/core/implementation/annotation/Host.java

```diff
@@ -41,7 +44,8 @@
  *  }</pre>
  */
 @Target(value = {TYPE})
-@Retention(RetentionPolicy.RUNTIME)        // Record this annotation in the class file and make it available during runtime.
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
The comment location suggests that this is for the host interface. I would put the comment above instead.